profiles.js

```diff
@@ -207,7 +207,7 @@ module.exports = function (options) {
             dojoId: parentDojo,
             userTypes: [userType]
           };
-          seneca.act({role: 'cd-dojos', cmd: 'save_usersdojos', userDojo: userDojo}, cb);
+          seneca.act({role: 'cd-dojos', cmd: 'save_usersdojos', userDojo: userDojo, user: args.user}, cb);
```

Comment:
we shoudn't get there if the user is an o13 as it's unlikely they'll have linked a parent, am i right?